doc/indices.sgm

```diff
@@ -8,15 +8,29 @@
         </title>
         <para>
           <application>pgSphere</application> uses <literal>GiST</literal>
-          to create spherical indices. An index speeds up the execution
-          time of operators <link
+          and Block Range INdexing (<literal>BRIN</literal>) algorithms to create
+          spherical indices.
+          <literal>GiST</literal> index represents the R-Tree implementation for
```

Comment:
Change "index represents the" to "indexes utilize an" on this line.

Reply:
No problem, I will change it. What do you think if to change 'R-**T**ree' to 'R-**t**ree'?

Reply:
> What do you think if to change 'R-Tree' to 'R-tree'?

Yes, good idea!